packages/web/src/components/Home/index.tsx

```diff
@@ -57,10 +57,13 @@ export default function Home() {
                         </div>
                     </div>
                     <div className={styles.videoWalkthrough}>
-                        <div className={styles.videoWalkthroughFake}>
-                            Video placeholder: Demo showing how to use the app via interactive
-                            walkthrough
-                        </div>
+                        <iframe
+                            src="https://player.vimeo.com/video/974452570?badge=0&amp;autopause=0&amp;player_id=0&amp;app_id=58479"
+                            width="267"
```

Comment:
This seems a little narrower than the mockup, though might just be on my screen

Reply:
Yeah - it couldn't be the same width and height so I chose the same height lmk if you think I should've done width instead (aspect ratio is why)

Reply:
Makes sense! I'd vote for whichever one makes it bigger overall